core-java-modules/core-java-arrays-operations-basic/README.md

```diff
@@ -0,0 +1,19 @@
+## Core Java Arrays - Basic Operations
+
+This module contains articles about basic operations on arrays in Java
```

Comment:
Let's try this:

```
This module contains articles about Java array fundamentals. They assume no previous background knowledge on working with arrays in Java.
```

I think listing specifics like iteration or retrieval is problematic since there are advanced ways to iterate, etc. I think the difference between basic and advanced is probably what the article expects you to already know before reading.